Flickr/Sources/FlickrAppVCs/PhotoViewerVC.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  PhotoViewerVC.swift
+//  Flickr
+//
+//  Created by Ananth Desai on 28/03/22.
+//
+
+import Foundation
+import Nuke
+import UIKit
+
+class PhotoViewerVC: UIViewController {
+    private var url: URL
+    private var favouriteState: Bool?
+    private var imageTitle: String?
+    private weak var stackView: UIStackView?
+    private weak var imageView: UIImageView?
+    private weak var favoriteEnabledButton: UIButton?
+    private weak var favoriteDisabledButton: UIButton?
+
+    init(url: URL, title _: String, imageTitle: String) {
```

Comment:
Why is `title` not used in the VC? And if there really is no use for it. Why not remove it as a parameter?